src/benchmarks/micro/corefx/System.Text.RegularExpressions/Perf.RegexRedux.cs

```diff
@@ -1,66 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.IO;
-using System.Threading.Tasks;
-using BenchmarkDotNet.Attributes;
-using MicroBenchmarks;
-
-namespace System.Text.RegularExpressions.Tests
-{
-    [BenchmarkCategory(Categories.CoreFX)]
-    public class RegexRedux
```

Comment:
Both this and RegexRedux_5 are creating new compiled Regex instances on every invocation; that adds non-trivial overhead:
https://github.com/dotnet/runtime/pull/109#issuecomment-555670867

Reply:
great catch! I've extended the regex-redux-5 benchmark to run for both RegexOptions.Compiled and RegexOptions.None